wallet/ledger/device.go

```diff
@@ -2,11 +2,18 @@ package ledger
 
 import (
 	"fmt"
+	"math/big"
 	"time"
 
+	"github.com/btcsuite/btcd/btcec"
```

Comment:
Please switch to `github.com/btcsuite/btcd/btcec/v2` (see the latest SDK that also switched).